app/assets/javascripts/pretty-text/addon/emoji.js

```diff
@@ -93,12 +93,13 @@ export function performEmojiUnescape(string, opts) {
       isReplacableInlineEmoji(string, index, opts.inlineEmoji);
 
     const title = opts.title ?? emojiVal;
+    const tabIndex = opts.tabIndex ? ` tabindex=${opts.tabIndex}` : "";
```

Comment:
Cool! This also needs a unit test in `...unit/lib/emoji-test.js` when using this new option.